### ISSUE TITLE
Set Up Encryption dialog doesn't mention that conversation keys will be backed up to server

### ISSUE THREAD

**Issue by notramo** [vector-im/riot-web#13396]
![image](https://user-images.githubusercontent.com/26059643/80356497-d4c01580-8879-11ea-9912-b025a4704c11.png)

Some users doesn't want to save encryption keys to the server. This dialog doesn't mention that it will save them to the server.
I suggest the following wording:
**Back up encrypted message keys to the server**

**Comment by andrewperry** [comment]
Yes, it is also not clear which server they will be backed up to - will it be at matrix.org, the home server, identity server....? What privacy policy applies on that server?

**Comment by andrewperry** [comment]
When I upgraded my Mac client to the latest version it seems to now be prompting to upgrade security, requesting my password, without clarity on what's going to happen.
